### PR TITLE
Use additional_role_tags instead of role_tags

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request changes the terraform code to pass `additional-role_tags` instead of `role_tags` to the [cisagov/s3-read-role-tf-module](https://github.com/cisagov/s3-read-role-tf-module) Terraform module.

## 💭 Motivation and context ##

This is necessary due the changes from cisagov/s3-read-role-tf-module#15.

## 🧪 Testing ##

I tested this code by applying the example code via `terraform apply` and verifying that it functioned as expected.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,12 +60,13 @@ module "example" {
...
+| additional\_role\_tags | Tags to apply to the IAM role that allows read-only access to the specified Terraform state, in addition to the provider's default tags. | `map(string)` | `{}` | no |
 | assume\_role\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows read-only access to the specified Terraform state.  Note that the first "%s" in this value will get replaced with the role\_name variable and the second "%s" will get replaced with the terraform\_account\_name variable. | `string` | `"Allow assumption of the %s role in the %s account."` | no |
 | assume\_role\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows read-only access to the specified Terraform state.  Note that the "%s" in this value will get replaced with the role\_name variable. | `string` | `"Assume%s"` | no |
+| aws\_region | The AWS region where the non-global resources are to be provisioned (e.g. "us-east-1"). | `string` | `"us-east-1"` | no |
```
Comment (mcdonnnj) [inline README.md]: This snuck in but I don't believe it belongs.

**Review by mcdonnnj — APPROVED** [review]
Makes sense and looks good. I did have one comment that requires fixing.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: Great catch!  See commit f0e8ca1.

**Review by dav3r — APPROVED** [review]
Outstanding! 🍨

**Review by mcdonnnj — APPROVED** [review]
Makes sense 👍

**Pull request merged**